bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWbb/Template/customizecards.dat

```diff
@@ -0,0 +1,6 @@
+set param_card mass 45 TEMPLATEMH03
+set param_card mass 35 TEMPLATEMH02
+set param_card mass 25 125
+set param_card DECAY 45 1.00000e-03
+set param_card DECAY 35 1.00000e-03
+set param_card DECAY 25 4.00000e-03
```

Comment:
The higgs boson mass is already set to 125 GeV by default in the model, I don't think it's needed (and the width should be automatically computed). Why is the h02 and h03 decay width always set to 0.001? Shouldn't this depend by your mass hypothesis? Is the BR computation/xsec taken from some reference theory paper?